### PR TITLE
redefine API to provide business data

### PR DESCRIPTION
**Pull request by EdwinBetanc0urt**
Greetings this is done for greater compatibility of the ADempiere Vue application in terms of service providers, managing a more standard return structure, as are the JSON structures.

Note: This new definition depends on the changes made to the grpc business data client library https://github.com/erpcya/gRPC-Data-Client/pull/25/

### CODE REVIEW

**Comment by EdwinBetanc0urt** [comment]
Note: This new definition depends on the changes made to the grpc business data client library : https://github.com/erpcya/gRPC-Data-Client/pull/26

File: src/components/ADempiere/Favorites/index.vue [hunk 1]
```diff
@@ -61,28 +59,10 @@ export default {
   },
   methods: {
     getFavoritesList() {
```
Comment (yamelsenih) [inline src/components/ADempiere/Favorites/index.vue]: Hi, tis is a bad form of resolve this, the search should be on dashboard instead

**Review by yamelsenih — CHANGES_REQUESTED** [review]
Please change all dashboard for old implementation

**Comment by yamelsenih** [comment]
Please resolve conflicts

File: src/components/ADempiere/Favorites/index.vue (hunk 1 above)
Comment (EdwinBetanc0urt) [reply src/components/ADempiere/Favorites/index.vue]: Hi, changed library side data conversion for dashboards and added missing attributes and fixed attribute errors.

https://github.com/erpcya/gRPC-Data-Client/pull/28

**Comment by EdwinBetanc0urt** [comment]
Added an update to the grpc library for Business Data.

https://github.com/erpcya/gRPC-Data-Client/pull/28

**Pull request merged**